lxd/storage/drivers/driver_zfs.go

```diff
@@ -40,10 +40,11 @@ type zfs struct {
 func (d *zfs) load() error {
 	// Register the patches.
 	d.patches = map[string]func() error{
-		"storage_create_vm":       d.patchStorageCreateVM,
-		"storage_zfs_mount":       d.patchStorageZFSMount,
-		"storage_create_vm_again": nil,
-		"storage_zfs_volmode":     d.patchStorageZFSVolMode,
+		"storage_create_vm":                        d.patchStorageCreateVM,
+		"storage_zfs_mount":                        d.patchStorageZFSMount,
+		"storage_create_vm_again":                  nil,
+		"storage_zfs_volmode":                      d.patchStorageZFSVolMode,
+		"storage_rename_custom_volume_add_project": nil,
```

Comment:
I remember you mentioning having tested zfs, but now I'm confused.

ZFS records the mountpoint of all its datasets, so since you're not running a `zfs set mountpoint=/a/b/c` type command here, what's updating that property?

Reply:
@stgraber I believe this is done by the ZFS driver's RenameVolume() function call here https://github.com/lxc/lxd/pull/6987/files/a41e9ff151a259ebd9d906c4a6b3fb79bb590c34..2ced1fbbe3f78136375f75c4585fe4d6fc69c591#diff-96460c002c4158677403774cc42e42ddR66